test/unit_Polygon.jl

```diff
@@ -236,7 +236,9 @@ for N in [Float64, Float32, Rational{Int}]
 
     # check that empty polygon (infeasible constraints) has no vertices (#918)
     P = HPolygon([HalfSpace(N[1, 1], N(0)), HalfSpace(N[-1, 0], N(-1)),
-        HalfSpace(N[0, -1], N(-1))])
+                  HalfSpace(N[0, -1], N(-1))])
+    @test vertices_list(P) == Vector{Vector{N}}()
+    P = HPolygon([HalfSpace(N[1, 0], N(0)), HalfSpace(N[-1, 0], N(-1))])
```

Comment:
can you add `isempty(P)` as a test?

Reply:
There is already such a test for an empty polygon. Since the result is an empty `Vector`, it will work (I tried).

Reply:
ok. i would add it because that the test in your linked issue.

Reply:
I added it.